intranet/apps/preferences/fields.py

```diff
@@ -10,22 +10,22 @@ class PhoneField(forms.Field):
         'invalid': 'Please enter a valid phone number.'
     }
 
-    def __init__(self, max_length=None, *args, **kwargs):
-        self.max_length = max_length
+    def __init__(self, *args, **kwargs):
         super(PhoneField, self).__init__(*args, **kwargs)
-        self.validators.append(validators.RegexValidator(r'^[0-9]+$', 'Please enter a valid phone number.'))
-        if max_length is not None:
-            self.validators.append(validators.MaxLengthValidator(int(max_length)))
+        self.validators.append(validators.RegexValidator(r'^\+?1?\d{9,15}$', 'Please enter a valid phone number.'))
 
     def to_python(self, value):
         "Returns a Unicode object."
         if value in self.empty_values:
-            return ''
+            return ""
         value = force_text(value).strip()
         return value
 
+    @staticmethod
+    def prepare_value(value):
+        return "" if value == "None" else value
+
     def widget_attrs(self, widget):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/tjhsst-ion/ion/file/2366796574/issues/source?bid=3212528&fileBranchId=3221078#l28)